meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb

```diff
@@ -9,8 +9,9 @@ DEPENDS = "qtbase qtdeclarative qtwebkit"
 BRANCH = "Intreprid_stable_branch"
 VERSION = "6.1"
```

Comment:
If you use SRCREV, you don't need to BRANCH and VERSION variables. Please remove those.